### ISSUE TITLE
Reader not opening for comics

### ISSUE THREAD

**Issue by abillauer** [ajslater/codex#393]
I am using codex version 1.6.3 on a docker. I installed codex on a fresh ubuntu install and imported the comics library with the "Import Metadata on Library Scan" unchecked.

When opening up individual books the gui flashes the front cover for a moment and then the reader displays "Book Not Found Close the reader and find another book - Try to load book again" 

Clicking reload or refreshing the browser doesn't help.

The log reads: 
```

2024-07-04 15:06:14 UTC WARNING Invalid arc {'group': 'f', 'pks': (8083,), 'filters': {}, 'name': 'COMIC_NAME'} submitted to reader for comic 60508.

2024-07-04 15:06:14 UTC WARNING Invalid arc {'group': 'f', 'pks': (8083,), 'filters': {}, 'name': 'COMIC_NAME'} submitted to reader for comic 60508.
```
Am I doing something wrong here?

Thanks!

**Comment by abillauer** [comment]
Ok. Something odd I noticed. When I try to read subsequent books I get the same error with the same title as the original error but a different number at the end.

WARNING Invalid arc {'group': 'f', 'pks': (8083,), 'filters': {}, 'name': '**COMIC_NAME** '} submitted to reader for comic 108799.

**Comment by ajslater** [comment]
Just got back from a little vacation. Your bug report is informative, thanks! I'll try to reproduce tomorrow.

**Comment by abillauer** [comment]
Thanks. Welcome back! 

Let me know if you need any additional information.

**Comment by ajslater** [comment]
This should be fixed in [Codex v1.6.4](https://github.com/ajslater/codex/releases/tag/v1.6.4)
Thanks again for reporting!
